src/views/EditSimple.vue

```diff
@@ -416,10 +448,28 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss" scoped>
 .event-popover__inner {
 	width: unset !important;
 	max-width: unset !important;
 	min-width: 300px !important;
 }
+.modal-mask {
+	position: fixed;
+	z-index: 9998;
+	//the height of header
```

Comment:
Please test this with an embedded calendar

Reply:
sure, how can i add/create an embed calender?

Reply:
looks good. Why did you specify this, did you expect to be broken? (Just to make sure i didnt miss something.)

Reply:
Because of the specific value used to position the mask relative to the header. If the header sizes are not identical, this solution wouldn't work.
Thank you for checking.